_posts/2019-09-13-dns-tips.md

```diff
@@ -0,0 +1,101 @@
+---
+title: "DNS tips"
+date: 2019-09-13T12:00:21+02:00
+draft: false
+categories: ["openshift", "kubernetes", "metal3", "dns", "tips"]
+author: Pedro Ibáñez Requena
+---
+
+## Prerequisites
+When preparing a baremetal installation in a user-provisioned DNS scenario it's important
```

Comment:
add a ,

Reply:
you also want to indicate you re talking about external dns requirements here

Reply:
Done, thx.